OpenMEEG/libs/OpenMEEGMaths/include/symmatrix.h

```diff
@@ -324,11 +324,11 @@ namespace OpenMEEG {
         // LU
         int *pivots = new int[nlin()];
         int Info = 0;
-        DSPTRF('U', nlin(), data(), pivots, Info);
+        DSPTRF('U', (int)nlin(), data(), pivots, Info);
         // Inverse
         int sz = (int) this->nlin() * 64;
         double *work = new double[sz];
-        DSPTRI('U', nlin(), data(), pivots, work, Info);
+        DSPTRI('U', (int)nlin(), data(), pivots, work, Info);
```

Comment:
max int in 32bits = 2'147'483'647
.... ok maybe we will have a matrix of 20'000 x 10'0000 one day...
I'll change that as we agreed into a function sizet_to_int